helper/strutil/strutil.go

```diff
@@ -174,6 +174,16 @@ func ParseArbitraryStringSlice(input string, sep string) []string {
 	return ret
 }
 
+// TrimStrings takes a slice of strings and returns a slice of strings
+// with trimmed spaces
+func TrimStrings(items []string) []string {
```

Comment:
OK. I definitely see both ways, some might not expect the original to be altered.